src/pages/Accounts/Account.tsx

```diff
@@ -0,0 +1,28 @@
+import { Flex } from '@chakra-ui/react'
+import { useParams } from 'react-router-dom'
+import { AssetAccountDetails } from 'components/AssetAccountDetails'
+import { Page } from 'components/Layout/Page'
+import { selectAssetByCAIP19 } from 'state/slices/assetsSlice/assetsSlice'
+import {
+  AccountSpecifier,
+  selectFeeAssetByAccountId
+} from 'state/slices/portfolioSlice/portfolioSlice'
+import { useAppSelector } from 'state/store'
+export interface MatchParams {
```

Comment:
```suggestion
export type MatchParams {
```

only use `interface` when something needs to be implemented or extended, use `type` otherwise